ui/shared/status/StatusGifColumn.qml

```diff
@@ -14,14 +16,36 @@ Column {
         id: repeater
 
         delegate: Rectangle {
-            height: animation.height
+            height: animation.status != Image.Ready ? loader.height : animation.height
             width: root.gifWidth
 
+            Rectangle {
+                id: loader
+                height: 100
+                width: root.gifWidth
+                visible: animation.status != Image.Ready
+                radius: Style.current.radius
+                rotation: 90
+                gradient: Gradient {
+                    GradientStop { position: 0.0; color: "transparent" }
+                    GradientStop { position: 1.0; color: "#E6EEF2" }
```

Comment:
Does this gradient color look okay in Dark Mode?

Reply:
With a few needed adjustement, now it does work